Source/QED/QuantumSyncDummyTable.H

```diff
@@ -0,0 +1,78 @@
+#ifndef WARPX_quantum_sync_dummy_tables_h_
+#define WARPX_quantum_sync_dummy_tables_h_
+
+#include "QuantumSyncEngineInnards.H"
+
+#include <AMReX_REAL.H>
+#include <AMReX_Gpu.H>
+
+#include <limits>
+
+namespace QedUtils{
+
+//A default mini-table used for test purposes
+const struct //QuantumSyncEngineInnardsDummy
+{
+    picsar::multi_physics::quantum_synchrotron_engine_ctrl<amrex::Real> ctrl{
+        0.001, //chi_part_min
+        0.001, //chi_part_tdndt_min
+        200, //chi_part_tdndt_max
+        64, //chi_part_tdndt_how_many
+        0.001, //chi_part_tem_min
+        200, //chi_part_tem_max
+        2, //chi_part_tem_how_many
+        2 //prob_tem_how_many
+    };
```

Comment:
Same comment as above.

Reply:
I've just implemented your solution. I agree that it could be safer.